From bc5e57550a8c6acd2fb13364e334d17b998845d3 Mon Sep 17 00:00:00 2001
From: Mrunali Nesarkar <mnesarka@qti.qualcomm.com>
Date: Wed, 13 Nov 2024 16:04:10 +0530
Subject: [PATCH 1/1] systemd: mask journald messages and remove rules with
 render

Mask the error messages from journald and also remove the udev
rules which sets the group as render as render group doesnt
exist in the build.

Signed-off-by: Abinaya P <abinayap@codeaurora.org>
---
 rules.d/50-udev-default.rules.in | 2 --
 src/core/exec-invoke.c           | 5 -----
 2 files changed, 7 deletions(-)

diff --git a/rules.d/50-udev-default.rules.in b/rules.d/50-udev-default.rules.in
index af4f594791..e1b6ea800e 100644
--- a/rules.d/50-udev-default.rules.in
+++ b/rules.d/50-udev-default.rules.in
@@ -53,8 +53,6 @@ SUBSYSTEM=="dvb", GROUP="video"
 SUBSYSTEM=="media", GROUP="video"
 SUBSYSTEM=="cec", GROUP="video"
 
-SUBSYSTEM=="drm", KERNEL=="renderD*", GROUP="render", MODE="{{GROUP_RENDER_MODE}}"
-SUBSYSTEM=="kfd", GROUP="render", MODE="{{GROUP_RENDER_MODE}}"
 SUBSYSTEM=="accel", GROUP="render", MODE="{{GROUP_RENDER_MODE}}"
 
 SUBSYSTEM=="misc", KERNEL=="sgx_enclave", GROUP="sgx", MODE="0660"
diff --git a/src/core/exec-invoke.c b/src/core/exec-invoke.c
index 28d6142318..8fb85ff44c 100644
--- a/src/core/exec-invoke.c
+++ b/src/core/exec-invoke.c
@@ -597,11 +597,6 @@ static int setup_output(
         case EXEC_OUTPUT_JOURNAL_AND_CONSOLE:
                 r = connect_logger_as(context, params, o, ident, fileno, uid, gid);
                 if (r < 0) {
-                        log_exec_warning_errno(context,
-                                               params,
-                                               r,
-                                               "Failed to connect %s to the journal socket, ignoring: %m",
-                                               fileno == STDOUT_FILENO ? "stdout" : "stderr");
                         r = open_null_as(O_WRONLY, fileno);
                 } else {
                         struct stat st;
-- 
2.34.1

